docs/configure/overview.md

```diff
@@ -26,7 +26,7 @@ The main configuration file is `main.js`. This file controls the behavior of the
 
 The `main.js` configuration file is a [preset](../addons/addon-types.md) and as such has a powerful interface, but the key fields within it are:
 
-- `stories` - an array of globs that indicates the [location of your story files](#configure-story-loading), relative to `main.js`.
+- `stories` - an array of globs that indicates the [location of your story files](#configure-story-loading), relative to `main.js`. Set the default story by placing it first in this array.
 - `addons` - a list of the [addons](/addons) you are using.
 - `webpackFinal` - custom [webpack configuration](./webpack.md#extending-storybooks-webpack-config).
 - `babel` - custom [babel configuration](./babel.md).
```

Comment:
@doublejosh we have a bit of room for improvement here. We probably could go with your changes as an aside. Turning this into:
```md
The `main.js` configuration file is a [preset](../addons/addon-types.md) and as such has a powerful interface, but the key fields within it are:

- `stories` - an array of globs that indicates the [location of your story files](#configure-story-loading), relative to `main.js`.
- `addons` - a list of the [addons](/addons) you are using.
- `webpackFinal` - custom [webpack configuration](./webpack.md#extending-storybooks-webpack-config).
- `babel` - custom [babel configuration](./babel.md).

<div class="aside">
Tip/Note: You can customize Storybook's default story by referencing it as the first item of  the `stories` array
</div>
```

Something in this nature. Let me know what you think.